test/jdk/com/sun/crypto/provider/KDF/HKDFExhaustiveTest.java

```diff
@@ -304,6 +306,9 @@ private static void testGetInstanceNegative() {
     Utils.runAndCheckException(
         () -> KDF.getInstance(KDF_ALGORITHMS[0], null, INVALID_STRING),
         NoSuchProviderException.class);
+    Utils.runAndCheckException(
+        () -> KDF.getInstance(KDF_ALGORITHMS[0], new TestKDFParams(), SUNJCE),
+        InvalidAlgorithmParameterException.class);
```

Comment:
How about also test the other `getInstance` method with `String provider` argument?